account_payment_mode_auto_reconcile/models/account_invoice.py

```diff
@@ -82,6 +84,16 @@ def auto_reconcile_credits(self, partial_allowed=True):
                     continue
                 invoice.assign_outstanding_credit(credit.get('id'))
 
+    @api.multi
+    def _filter_payment_same_journal(self, credits):
```

Comment:
```suggestion
    def _filter_payment_same_journal(self, credits_dict):
```
:warning: Python constant: https://docs.python.org/3/library/constants.html#credits